README.md

```diff
@@ -79,7 +97,12 @@ Imagine that you take the code from this question, save it to a file called `foo
 
 What would be the output? Explain your answer.
 
-<!-- Replace this comment with your answer -->
+sushi is tasty
+
+It outputs this because howMuchILikeSushi >= 3 is false, given 2 >= 3 is false. Thus, the first if block is not executed.
+
+However howMuchILikeSushi > 0 is true, given 2 > 0 is true. Thus the second if block is executed.
+
```

Comment:
Oops, missed this on my first pass. Sorry, this is actually incorrect, and the reason is that `x` is undefined. This code will actually throw an error.